setup.py

```diff
@@ -39,6 +39,8 @@
         'pygments>=2.2.0,<3.0',
         'wagtail==2.2.*',
         'wagtailfontawesome>=1.1.3,<2.0',
+        'geocoder==1.38.1',
```

Comment:
Would it be safe to use <2.0?

Reply:
That's for us to decide I suppose.  I'm not sure what our dependency policy is.

Reply:
Go with `>=1.38.1,<2.0`